src/plugins/workspace/public/workspace_client.ts

```diff
@@ -118,7 +120,20 @@ export class WorkspaceClient {
     });
 
     if (result?.success) {
-      this.workspaces.workspaceList$.next(result.result.workspaces);
+      const resultWithWritePermission = await this.list({
+        perPage: 999,
+        permissionModes: [WorkspacePermissionMode.LibraryWrite],
```

Comment:
To support `permissionModes` parameters, I think we need to modify `src/plugins/workspace/server/workspace_client.ts` and pass `ACLSearchParams:{permissionModes}` to saved object find method.

Reply:
Sure, I will add `ACLSearchParams: { permissionModes: options.permissionModes }` to find object in the `src/plugins/workspace/server/workspace_client.ts`.